documentation/notes/joss_paper/paper.md

```diff
@@ -68,12 +67,12 @@ However, these models are less often considered in literature, despite their dem
 For an efficient handling of the model by users, its documentation is conceived such that its equations and parameters are explained and linked to the code.
 In the future, its development will be done in a modular fashion which enable the connection of the atmosphere to various other subsystems and its usage with built-in and external toolboxes.
 
-The choice to use Python was specifically made to facilitate its use in Jupyter [@jupyter] notebooks and the multiple recent machine learning libraries that are available in this 
+The choice to use Python was specifically made to facilitate its use in Jupyter [@jupyter] Notebooks and the multiple recent machine learning libraries that are available in this
```

Comment:
Formally it is Notebook with a capitalised 'N', as used throughout the Jupyter official pages and documentation (e.g. see [here](https://jupyter-notebook.readthedocs.io/en/stable/index.html)).

Reply:
Ok. Noted.